update-doi-request/src/test/java/no/unit/nva/doi/requests/handlers/ApiUpdateDoiRequestHandlerTest.java

```diff
@@ -248,20 +249,36 @@ private Publication readPublicationDirectlyFromDynamo(UUID identifier) throws No
         return doiRequestsService.fetchDoiRequestByPublicationIdentifier(identifier).orElseThrow();
     }
 
-    private Publication expectedDoiRequestSummary(Publication originalPublication,
-                                                  Publication updatedPublication) {
-        var includedDoiRequest = new DoiRequest.Builder()
-            .withDate(mockNow)
+    private Publication expectedUpdatedPublication(Publication originalPublication,
+                                                   Publication updatedPublication) {
+        //modified dates need to be copied from the actual updated publication because the modifiedDate
+        //cannot be controlled
+
+        DoiRequest expectedDoiRequestWithUnsyncedModifiedDate = new DoiRequest.Builder()
+            .withCreatedDate(mockNow)
             .withStatus(DoiRequestStatus.APPROVED)
             .build();
 
-        return
-            originalPublication
-                .copy()
-                .withDoiRequest(includedDoiRequest)
-                // copy actual modified date because it is set in Publication class by uncontrollable clock.
-                .withModifiedDate(updatedPublication.getModifiedDate())
-                .build();
+        Publication expectedPublicationWithWrongDates = originalPublication
+            .copy()
+            .withDoiRequest(expectedDoiRequestWithUnsyncedModifiedDate)
+            .build();
+
+        return syncUnControllableDates(expectedPublicationWithWrongDates, updatedPublication);
+    }
+
+    // copy actual modified dates because they are set in Publication class by uncontrollable clock.
```

Comment:
nit: Might be worth having this as @return in proper javadoc, even if private method since we are commenting it anyway.

Reply:
Unsure if its wroth saying it only copies the modification timestamps? 
I guess the real fix is to be able to inject the clock for more controlled testing, which we can `tick` forward ourselves in the tests and assert against? 

Good enough for now.